book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
we need to make rentals read/write, because we are trying to model the many-to-many relationship between Person and Book using the intermediate class Rental, right now it's set to only read 

- [ ] Please update the setter for rentals